apps/drain-safe/src/utils/gateway.ts

```diff
@@ -0,0 +1,26 @@
+import { fetchJson } from '.';
+
+export interface Asset {
+  balance: string;
+  ethValue: string;
+  timestamp: string;
+  fiatBalance: string;
+  fiatConversion: string;
+  fiatCode: string;
+  tokenAddress: string | null;
+  token: null | {
+    name: string;
+    symbol: string;
+    decimals: number;
+    logoUri: string;
+  };
+}
+
+export const CURRENCY = 'USD';
+
+export async function fetchSafeAssets(safeAddress: string, safeNetwork: string): Promise<Asset[]> {
+  const network = safeNetwork.toLowerCase(); // mainnet, rinkeby etc
+  const url = `https://safe-transaction.${network}.gnosis.io/api/v1/safes/${safeAddress}/balances/${CURRENCY.toLowerCase()}/?exclude_spam=true`;
```

Comment:
You have to be aware that on some networks this might not be available, so in that case the request will fail ;)

Minor nitpick: the file is called `gateway.ts` but we use don't use the gateway here :P

Reply:
Renamed 👍